### PR TITLE
1173-documentation-add-back-the-updated-examples

### PR DESCRIPTION
**Pull request by SandyEma9**

### CODE REVIEW

**Comment by SandyEma9** [comment]
@robert-altom @RobertPoienar @gomboskriszta15 @iulianatodoran @MihaiCuc98 @ReichEdvin

File: Docs/source/pages/examples.md [hunk 1]
```diff
@@ -37,14 +35,14 @@ We plan to add more examples in the near future.
 
 * C# tests [SampleScenes](https://alttester.com/app/uploads/AltTester/sdks/SampleScene.unitypackage)
```
Comment (iulianatodoran) [inline Docs/source/pages/examples.md:36]: We need also to add the SampleScenes package back

File: Docs/source/pages/examples.md [hunk 2]
```diff
@@ -37,14 +35,14 @@ We plan to add more examples in the near future.
...
 
-    Import this package under your AltTester package in Unity, to have access to the SampleScenes and C# tests.
+    Import this package under your AltTester package in Unity, to have access to the SampleScenes and C# tests.-->
 
 
-**3.** Example test projects for AltTester Unity SDK v.2.0.0 and the New Input System:
+**2.** Example test projects for AltTester Unity SDK v.2.0.1 and the New Input System:
 
 * C# tests [CoinCollector](https://github.com/alttester/EXAMPLES-NewInputSystem--CoinCollector-CSharp)
```
Comment (iulianatodoran) [inline Docs/source/pages/examples.md:43]: This example was also updated
```suggestion
* C# tests [CoinCollector](https://github.com/alttester/EXAMPLES-NewInputSystem--CoinCollector-CSharp)
```

File: Docs/source/pages/examples.md [hunk 3]
```diff
@@ -37,14 +35,14 @@ We plan to add more examples in the near future.
...
 * C# tests [LeanTouch](https://github.com/alttester/EXAMPLES-NewInputSystem--LeanTouch-CSharp)
```
Comment (iulianatodoran) [inline Docs/source/pages/examples.md:44]: This example was also updated

```suggestion
* C# tests [LeanTouch](https://github.com/alttester/EXAMPLES-NewInputSystem--LeanTouch-CSharp)
```

**Review by iulianatodoran — APPROVED** [review]

**Review by rucindrea — APPROVED** [review]

**Pull request merged**